stratum/hal/lib/tdi/tofino/BUILD

```diff
@@ -103,7 +104,7 @@ stratum_cc_library(
 stratum_cc_library(
     name = "tofino_chassis_manager",
     srcs = ["tofino_chassis_manager.cc"],
-    hdrs = ["tofino_chassis_manager.h"],
+    hdrs = ["tofino_chassis_manager.h", "tofino_port_manager.h"],
```

Comment:
Style note: when there are multiple targets in a list, they're usually written one per line (with trailing commas):
```
hdrs = [
    "tofino_chassis_manager.h",
    "tofino_port_manager.h",
],
```